packages/webpack-plugin/index.js

```diff
@@ -0,0 +1,150 @@
+// @ts-check
+
+const path = require('path')
+const fs = require('fs/promises')
+
+// when client is bundled this gets its output path
+// regex works both on escaped and non-escaped code
+const prismaDirRegex =
+  /\\?"?output\\?"?:\s*{(?:\\n?|\s)*\\?"?value\\?"?:(?:\\n?|\s)*\\?"(.*?)\\?",(?:\\n?|\s)*\\?"?fromEnvVar\\?"?/g
+
+async function getPrismaDir(from) {
+  // if we can find schema.prisma in the path, we are done
+  if (await fs.stat(path.join(from, 'schema.prisma')).catch(() => false)) {
+    return from
+  }
+
+  // otherwise we need to find the generated prisma client
+  return path.dirname(require.resolve('.prisma/client', { paths: [from] }))
+}
+
+// get all required prisma files (schema + engine)
+async function getPrismaFiles(from) {
+  const prismaDir = await getPrismaDir(from)
+  const filterRegex = /schema\.prisma|.*?engine.*?/
```

Comment:
isn't it equivalent to this:
```suggestion
  const filterRegex = /schema\.prisma|.*engine.*/
```
or am I missing something and `?` is necessary here?

Reply:
Ah, no, I'm wrong, they are quite a bit different.

That said, `.*?engine.*?` only matches `libquery_engine` in `libquery_engine-debian.so.node` or `query-engine` in `query-engine-debian`. Is this expected?

Reply:
Good catch, my only intent was just to check if it matched. So I think `/engine/` should be enough.

Reply:
```suggestion
  const filterRegex = /schema\.prisma|engine/
```